src/python/pants_release/generate_github_workflows.py

```diff
@@ -879,7 +886,11 @@ def build_wheels_job(
             "steps": [
                 *initial_steps,
                 install_protoc(),  # for prost crate
-                *([] if platform == Platform.LINUX_ARM64 else [install_go()]),
+                *(
+                    [install_python_headers_in_manylinux_container()]
+                    if platform == Platform.LINUX_ARM64
```

Comment:
I presume this builds in the x86_64 containers because the package happens to already be installed?

Reply:
I guess so, some small difference between the two images.